Tests/AlgoliaSearch/ReindexCommandTest.php

```diff
@@ -83,7 +86,7 @@ public function reIndex($use_entity_alias, $safe = true)
 
         $input = new ArrayInput($options);
 
-        $this->command->run($input, new ConsoleOutput());
+        $this->command->run($input, new NullOutput());
```

Comment:
I've changed it to `NullOutput` to avoid output to the console during test execution which may cause risky tests (at least if [configured](https://phpunit.de/manual/current/en/risky-tests.html#risky-tests.output-during-test-execution)). But if it's breaking the build I'll change for sure (unfortunately it's not breaking the build on my machine 😟)! WDYT of using the `BufferedOutput` which could then also be used to check if some excepted output has been written?